docs/configuration.md

```diff
@@ -332,6 +334,18 @@ Overrides the number of automatically determined test shards for Flank to use. U
     testShards.set(5)
     ```
 
+### maxTestShards
+The maximum number of shards.
```

Comment:
can we also add a deprecation warning to the testShards property?

Reply:
I've added annotation, but I guess that's not enough 😄 
Actually, I was thinking about how to solve it. Custom getter with printing message when used? I would be grateful for any ideas

Reply:
Ah i meant in the documentation as well so that it appears on https://runningcode.github.io/fladle/configuration/